pkg/work/hub/controllers/manifestworkreplicasetcontroller/manifestworkreplicaset_index.go

```diff
@@ -71,7 +72,11 @@ func (m *ManifestWorkReplicaSetController) manifestWorkQueueKeyFunc(obj runtime.
 	if !ok {
 		return ""
 	}
-	return key
+	keys := strings.Split(key, ".")
```

Comment:
why do we need this?

Reply:
the manifestwork has a reference to manifestworkreplicaset by a label value of <namespace>.<name>
we cannot use "/" here since it is invalid for label value.